test/InterchainTokenService.js

```diff
@@ -84,6 +87,36 @@ describe('Interchain Token Service', () => {
         return [token, tokenManager, tokenId];
     };
 
+    deployFunctionsTest.lockUnlock = async function deployNewLockUnlock(
```

Comment:
Note: There are multiple `async function deployNewLockUnlock` and `async function deployNewGateway` functions defined, need to refactor the code. (Would be a separate PR)

Reply:
I generally prefer to avoid making code quality worse temporarily to fix it later. Instead, we should try adding the test in a cleaner manner even though it might take more work than originally expected for the task.